sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -852,6 +852,27 @@ class DataFrame private[sql](
   }
 
   /**
+   * Returns a new [[DataFrame]] with a column dropped.
+   * @group dfops
+   */
+  def drop(col: String, cols: String*): DataFrame = {
```

Comment:
Let's support dropping one column for now, and we can add other things in the future. We need to be extra careful when we add varargs, because they make it very hard to extend.